pkgs/development/python-modules/impacket/default.nix

```diff
@@ -1,6 +1,8 @@
 { lib
 , buildPythonPackage
 , chardet
+, charset-normalizer
```

Comment:
Looks like `chardet` can be removed according to https://github.com/fortra/impacket/pull/1359.